src/workflow-engine/charms/temporal-worker/oci_factory/notification/mattermost_notifier.py

```diff
@@ -0,0 +1,99 @@
+import json
+import logging
+import os
+import requests
+import time
+from typing import Optional
+
+
+COLORS = {
+    "success": "#33CC33",
+    "failure": "#CC0000",
+    "unknown": "#B0B0B0",
+}
+
+channel_id = os.environ.get(
+    "MATTERMOST_CHANNEL_ID", "yax1c3pmm7natd5ebw3cngyrpw"
+)  # `ROCKS Bot Test` by default
+mattermost_server = "https://chat.canonical.com"
```

Comment:
can you pls make these secrets and fix the channel ID?

Reply:
If you think this is a better idea, we can make both secrets and define them in the `.env` file.

Reply:
Resolved in [a0dec2e](https://github.com/canonical/oci-factory/pull/205/commits/a0dec2edef902c5887971b9930fb7797fa581ade)